nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/blobholder/SimpleBlobHolder.java

```diff
@@ -58,6 +59,11 @@ protected void init(List<Blob> blobs) {
         creationDate = Calendar.getInstance();
     }
 
+    @Override
+    public void setBlob(Blob blob) {
+        init(new ArrayList<>(Arrays.asList(blob)));
```

Comment:
Why not `init(List.of(blob)` since this is master?
Or do we need a mutable list?

Reply:
I'm not certain if we do but for anything that requires backports I'm using Java 8 syntax and being conservative with semantics (the existing constructor uses a mutable list).

Reply:
OK, right about keeping semantics.

Though, I think we could have a wider discussion about using or not Java 11 features/syntax on master for issues that require backports. I think that in some cases we should allow us to use different code, to improve master. But this case is not a good example for sure.